day_6/questions.md

```diff
@@ -2,12 +2,30 @@
 
 1. In your own words, what is a Class?
 
+A class a set or collection of items that all share the same characteristics. Members of the class can be described in the same way.
+
 1. In relation to a Class, what is an attribute?
 
+An attribute is a category within a class that can be applied to each member of the class. They can all be different attributes but from the same category.
+
 1. In relation to a Class, what is behavior?
 
+A behavior allows the attributes to be modified based on the type of data you want to manipulate. Behaviors can be applied to any or all objects of a class.
+
 1. In the space below, create a Dog class with at least 2 attributes and 2 behaviors
 
+Class Dog  
+Attributes:  
+hair_length  
+number_of_paws  
+Methods:  
+cut_hair  
+remove_paw
+
 1. How do you create an instance of a class?
 
+Once your class has been created and initialized, you set the name of your instance equal to the class name followed by .new.
+
 1. What questions do you still have about classes in Ruby?
+
+My question about classes in Ruby is how to decide if your class is too general and when to break it down into multiple classes.
```

Comment:
This is part of the art of programming. How one designs their program and separates concerns. It will generally change as your needs change. and you will learn some fundamental patterns of how to do this during your time at Turing. Some questions to keep in mind when designing. Is it easy to change my class? Does my class depend on other classes (hopefully not at all or very little). Does my class handle one concern or one chunk of functionality within my program (you want the answer to be yes).